core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -24,6 +24,29 @@ var ExplorationEditorTranslationTab = function() {
   var translationWelcomeModal = element(
     by.css('.protractor-test-translation-tab-welcome-modal'));
 
+  this.saveUploadedAudioButton = element(
+    by.css('.protractor-test-save-button'));
+
+  this.wrongFileTypeErrorMessage = element(by.css('div.error-message')).getText();
+
+  this.colorOfInitNode = function() {
+    return (element.all(by.css(
+      'rect.protractor-test-node-background.init-node')).last().
+      getCssValue('fill'));
+  };
+
+  this.colorsOfNormalNode = function(index) {
```

Comment:
Function names should start with a verb. Also, where is index used?
```suggestion
  this.getColorOfNormalNodes = function(index) {
```

Reply:
done

Reply:
@nithusha21 Actually, I was thinking of using index to get the normal node at specific index. But then I ended up using ```then``` in ExplorationTranslationEditor.js and forgot removing this purposeless index. Thanks a lot for pointing out. :)